proxy/nginx.conf

```diff
@@ -2,39 +2,36 @@ events {}
 
 http {
```

Comment:
- https://www.owasp.org/index.php/SCG_WS_nginx
- https://www.upguard.com/blog/how-to-build-a-tough-nginx-server-in-15-steps

Plus google "nginx hardening"

Reply:
Nadal brakuje podstawowych ustawień z https://www.owasp.org/index.php/SCG_WS_nginx:
- Remove Version number
- Buffer Overflow Protection
- Mitigating Slow HTTP DoS Attack
- SSL/TLS Configuration - Tutaj ograniczyłbym się TLSv1.2 - obecnie oznacza to że blokujemy anadroidy starsze niż 5 https://en.wikipedia.org/wiki/Transport_Layer_Security#Web_browsers
- HTTP secure Header
- Najpewniej będziesz musiał dodac trochę header-ów żeby proxy działało poprawnie z takimi rzeczami jak CSRF: https://docs.djangoproject.com/en/2.2/ref/settings/#secure-proxy-ssl-header
  - Przykład z którejś z moich konfiguracji:
```
    location / {
        proxy_set_header    Host                    $http_host;
        proxy_set_header    X-Real-IP               $remote_addr;
        proxy_set_header    X-Forwarded-For         $proxy_add_x_forwarded_for;
        proxy_set_header    X-Forwarded-Host        $http_host;
        proxy_set_header    X-Forwarded-Server      $host;
        proxy_set_header    X-Forwarded-Port        ${ADMIN_PORT};
        proxy_set_header    X-Forwarded-Proto       $scheme;
        proxy_set_header    X-Forwarded-Protocol    $scheme;

        proxy_pass          http://bpap_django:8000;
        proxy_read_timeout  90;
    }
```